scripts/device-test.ps1

```diff
@@ -76,7 +76,7 @@ try
         if (!(Get-Command xharness -ErrorAction SilentlyContinue))
         {
             Push-Location ($CI ? $env:RUNNER_TEMP : $IsWindows ? $env:TMP : $IsMacos ? $env:TMPDIR : '/temp')
-            dotnet tool install Microsoft.DotNet.XHarness.CLI --global --version '10.0.0-prerelease*' `
+            dotnet tool install Microsoft.DotNet.XHarness.CLI --global --version '10.0.0-prerelease.25330.2' `
```

Comment:
Why `prerelease`?  It was bumped from `9` to `10` with the .NET 9 support PR in https://github.com/getsentry/sentry-dotnet/pull/3699. Can it jsut be `10.0.0`?